.github/workflows/dbld-images.yml

```diff
@@ -0,0 +1,43 @@
+name: Compile dbld-images
+
+on:
+  pull_request:
+  push:
+    paths:
+      - .github/workflows/dbld-images.yml
+      # List is from dbld/rules cache-image-% "WATCHED_FILES"
+      - "dbld/**"
+      - "packaging/rhel/syslog-ng.spec"
+      - "packaging/debian/control"
+  schedule:
+    - cron: '00 03 * * *'
+  workflow_dispatch:
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+  
+    steps:
+      - name: Checkout source code
+        uses: actions/checkout@v2
+
+      - name: Build the images
+        run: dbld/rules images -j $(nproc)
```

Comment:
Does this handle the rebuild of dependent images correctly (the build order might be important)?